packages/serializer/src/index.js

```diff
@@ -339,7 +342,16 @@ const jsxBlock = {
   matchMdast: (node, index, parent) =>
     node.type === 'jsx' && parent && parent.type === 'root',
   fromMdast: (node, index, parent, { visitChildren }) => {
-    const data = parseJSXBlock(node.value)
+    let data = {}
+    if (node.children) {
+      data = {
+        type: getComponentName(node.children[0].value),
+        props: {}
```

Comment:
Is there no way to get the outer block props at the moment?

Reply:
Not until we transpile the MDAST we're working with to JSX (since right now these interleaved blocks don't have JSX children like standard JSX blocks).